docs/development/releasing.rst

```diff
@@ -175,26 +171,28 @@ packages that use the full bugfix/maintenance branch approach.)
 
 #. Push up all these changes to the `astropy core repository`_::
 
-      $ git push --tags upstream v<version>
+      $ git push --tags upstream v<version branch>.x
 
    .. note::
 
       You may need to replace ``upstream`` here with ``astropy`` or
       whatever remote name you use for the `astropy core repository`_.
 
-#. If this is a release of the current release (not an LTS), update the
-   "stable" branch to point to the new release::
+#. If this is a release of the current release (i.e., not an LTS supported along
+   side a more recent version), update the "stable" branch to point to the new
+   release::
 
       $ git checkout stable
       $ git reset --hard v<version>
       $ git push upstream stable --force
```

Comment:
I think I've figured it out. Whenever I add and fetch a new remote from people who forked the repo when the "stable" tag was still around, it gets downloaded again. Since it's now deleted on the astropy repo (again), they may get rid of it when doing a ``git fetch --all --tags --prune``, but I'm not sure how many people are actually using the ``--tags --prune`` option combination. So probably we will have this issue in the foreseeable future.